eox_core/api/v1/tests/integration/test_views.py

```diff
@@ -370,6 +431,595 @@ def test_update_user_in_tenant_user_not_found(self, param: str, value: str) -> N
         )
 
 
+@ddt.ddt
+class TestEnrollmentAPIIntegration(BaseAPIIntegrationTest, UsersAPIRequestMixin, EnrollmentAPIRequestMixin):
+    """Integration test suite for the Enrollment API"""
+
+    def setUp(self) -> None:
+        """Set up the test suite"""
+        self.course_id = settings["COURSE_ID"]
+        self.mode = "audit"
+        super().setUp()
+
+    @ddt.data("email", "username")
+    def test_create_enrollment_success(self, param: str) -> None:
+        """
+        Test creating a enrollment with a valid user, course and mode in a tenant.
+
+        Open edX definitions tested:
+        - `get_edxapp_user`
+        - `create_enrollment`
+        - `check_edxapp_account_conflicts`
+        - `check_edxapp_enrollment_is_valid`
+
+        Expected result:
+        - The status code is 200.
+        - The enrollment is created successfully in the tenant with the provided data.
```

Comment:
For this to be true, we'd have to check for the new enrollment in the database, which is more or less the `test_get_enrollment_success` test case. So, I'd modify this to be: 

```suggestion
        - The response indicates the enrollment was created successfully. 
```